dojo/jira_link/helper.py

```diff
@@ -807,6 +814,14 @@ def update_epic(engagement):
 @task
 @dojo_model_from_id(model=Engagement)
 def add_epic(engagement):
+    add_epic_sync(engagement)
+
+
+# This code was not being tested, but calling it
+# as a task was not working. Using block execution
+# would not either here because this method is called
+# explicitly instead of through a save function
+def add_epic_sync(engagement):
     logger.info('trying to create a new jira EPIC for %d:%s', engagement.id, engagement.name)
```

Comment:
might be better ad debug level?

Reply:
No I don't think so. The Jira integration is very error prone to set up the first time. Not knowing if one of the functions even ran is extremely annoying. Most people look to the logs to figure out what happened and then will get nothing out of them.  Logging is not set to debug for most prod instances and knowing dojo well enough to know that enabling debugging in the setting is necessary is not common either.

Reply:
I think this indicates that we should do a better job somehow in the UI. I think we already do by posting alerts with the error messages.

Reply:
Okay well I'll set it to debug  on this PR because this is getting out of scope.